### ISSUE TITLE
getAccessToken does not seem to have access to user from GetUser

### ISSUE THREAD

**Issue by memelet** [oauthjs/node-oauth2-server#131]
In implementing generateToken() for JWT I see that getUser is invoked first, and the user is put on the Grant instance. But when generateToken is invoked there is no access to the user. Would it make sense for generateToken to be passed the Grant?

It would also be nice to have access to the client so expiration can be pulled from there.

**Comment by memelet** [comment]
I am fully unblocked using master. Thanks much!

**Comment by marioatlp** [comment]
Could you please do a version bump(npm publish) to include this change. I would really like to also be doing generation based on a user/client.
Thanks in advance.

**Comment by thomseddon** [comment]
Yes! Just want to merge last few things
